test/test_integration.rb

```diff
@@ -193,12 +193,12 @@ def test_phased_restart_via_pumactl
         done = true
       end
     end
-    # Stop
+
+    assert File.exist? @bind_path
```

Comment:
@Fudoshiki Why did you originally insert this assertion _after_ the stop command is issued? Shouldn't stopping the server remove the binder?

Reply:
We don't need this `assert` now

Reply:
Don't we? The original issue was that this file was removed after a phased restart was begun.

Reply:
- If we call `phased-restart` - the file must exist, connections from old workers should close
- If we stopping the server, the file must be deleted

Maybe we need move this in `until` block above